doc/nrf/ug_nrf91_features.rst

```diff
@@ -160,6 +160,213 @@ The :ref:`nrfxlib:nrf_modem` is released as an OS-independent binary library in
+
+* :ref:`http_full_modem_update_sample` sample - performs a full firmware OTA update of the modem.
+* :ref:`http_modem_delta_update_sample` sample - performs a delta OTA update of the modem firmware.
+* :ref:`http_application_update_sample` sample - performs a basic application FOTA update.
+* :ref:`aws_fota_sample` sample - performs a FOTA update via MQTT and HTTP, where the firmware download is triggered through an AWS IoT job.
+* :ref:`azure_fota_sample` sample - performs a FOTA update from the Azure IoT Hub.
+* :ref:`asset_tracker_v2` application - performs FOTA updates of the application, modem (delta), and boot (if enabled). It also supports nRF Cloud FOTA as well as AWS or Azure FOTA. Only one must be configured at a time.
+
+.. _nrf9160_ug_gnss:
+
+GNSS
+****
+
+An nRF9160-based device is a highly versatile device that integrates both cellular and GNSS functionality.
+Note that GNSS functionality is only available on the SICA variant and not on the SIAA or SIBA variants.
+See `nRF9160 SiP revisions and variants`_ for more information.
+
+There are many GNSS constellations (GPS, BeiDou, Galileo, GLONASS) available but GPS is the most mature technology.
+An nRF9160-based device supports both GPS L1 C/A (Coarse/Acquisition) and QZSS L1C/A at 1575.42 MHz.
+This frequency band is ideal for penetrating through layers of the atmosphere (troposphere and ionosphere) and suitable for various weather conditions.
+GNSS is designed to be used with a line of sight to the sky.
+Therefore, the performance is not ideal when there are obstructions overhead or if the receiver is indoors.
+
+The GNSS operation in an nRF9160-based device is time-multiplexed with the LTE modem.
+Therefore, the LTE modem must either be completely deactivated or in `RRC idle mode`_ or `Power Saving Mode (PSM)`_ when using the GNSS receiver.
+See the `nRF9160 GPS receiver Specification`_ for more information.
+Customers who are developing their own hardware with the nRF9160 are strongly recommended to use the `nRF9160 Antenna and RF Interface Guidelines`_ as a reference.
+See `GPS interface and antenna`_ for more details on GNSS interface and antenna.
+
+.. note::
+
+   Starting from |NCS| v1.6.0 (Modem library v1.2.0), the :ref:`GNSS socket <nrfxlib:gnss_extension>` is deprecated and replaced with the :ref:`GNSS interface <gnss_interface>`.
+
+
+Obtaining a fix
+===============
+
+GPS provides lots of useful information including 3D location (latitude, longitude, altitude), time, and velocity.
+
+The time to obtain a fix (also referred to as Time to First Fix (TTFF)) will depend on the time when the GPS receiver was last turned on and used.
+
+Following are the various GPS start modes:
+
+* Cold start - GPS starts after being powered off for a long time with zero knowledge of the time, current location, or the satellite orbits.
+* Warm start - GPS has some coarse knowledge of the time, location, or satellite orbits from a previous fix that is around 37 minutes old.
```

Comment:
Should this be "more than around 37 minutes old"? It should be more than 37 minutes old, otherwise it's a hot start. :)

Reply:
You are correct, my mistake 👍